README.md

```diff
@@ -69,18 +70,18 @@ var body: some View {
 }
 ```
 
-And just like that, you've got a workflow! You can now add more items to it or re-order the items that are there. To understand more of how this works and what is there [check out our developer docs.](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
+And just like that, you've got a workflow! You can now add more items to it or reorder the items that are there. To understand more of how this works, [check out our developer docs.](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
 
-# Check out our example apps
-We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They're even tested so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
+# Try Our Example Apps
+We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They've already been tested, so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
 
-# [See this to learn more](https://wwt.github.io/SwiftCurrent/Creating%20Workflows.html)
+# [See These to Learn More](https://wwt.github.io/SwiftCurrent/Creating%20Workflows.html)
```

Comment:
This may alter the meaning a bit too much. The link this takes you to should lead you to all kinds of documentation, not just what is bulleted below.

Reply:
Resolving this conversation since it's duplicated below in mine.